Add Humans of OTel blog post. (#5814)
open-telemetry/opentelemetry.io: content/en/blog/2024/humans-of-otel-na-2024.md

---
title: The Humans OpenTelemetry - KubeCon NA 2024
linkTitle: Humans of OTel NA 2024

> date: 2024-12-17

Reviewer: We'll probably want to set this to at least the 18th. I'll let @svrnm decide :)

Author: @svrnm I'm okay if this goes out Thursday or Friday. Ideally I'd like to get it out before the holidays.

Reviewer 2: Then let's try to get it out today :-)

Reviewer 2: ```suggestion
date: 2024-12-19
```

author: >-

>   [Adriana Villela](https://github.com/avillela) (Dynatrace),

Reviewer: ```suggestion
  [Adriana Villela](https://github.com/avillela) (Dynatrace)
```

---

We're back with our third edition of
[Humans of OpenTelemetry](/blog/2024/humans-of-otel-eu-2024/), this time from
KubeCon NA in Salt Lake City, Utah, USA. Once again,
[Reese Lee](https://github.com/reese-lee) and I interviewed OpenTelemetry
contributors and end users (and each other!), and learned how they got involved
with OTel:

- [Hazel Weakly (The Nivenly Foundation)](https://github.com/hazelweakly)
- [Eromosele Akhigbe (Sematext)](https://github.com/AkhigbeEromo)
- [Budha Bhattacharya (Tyk)](https://github.com/hellobudha)
- [Miguel Luna (Elastic)](https://github.com/mlunadia)
- [Adriana Villela (Dynatrace)](https://github.com/avillela)
- [David Gohberg (Monday)](https://github.com/dattto)
- [Endre Sara (Causely)](https://github.com/esara)
- [Braydon Kains (Google)](https://github.com/braydonk)
- [Christos Markou (Elastic)](https://github.com/ChrsMark)
- [Reese Lee (New Relic)](https://github.com/reese-lee)

Also, special thanks to:

- [Reese Lee](https://github.com/reese-lee), my co-interviewer
- [Henrik Rexed](https://github.com/henrikrexed) for providing the audio and
  video recording equipment, and doing the initial edits of the raw footage

> You can watch the full recording here:

Reviewer 2: ```suggestion
## Vide

You can watch the full recording here:
```

you might want to add a section title above the video as well so if someone scrolls down in the transcript they can use that link on the right menu to go back to the video

{{<youtube TIMgKXCeiyQ>}}

<br/>Thanks to everyone who has contributed to OpenTelemetry to date, and we
look forward to your continued contributions in 2025 and beyond! 🎉

## Transcript

> If reading is more your thing, check out the transcript of our conversations
> below.

Reviewer: ```suggestion
If reading is more your thing, the transcript of our conversations are below.
```
